core/federated/RTI/rti_common.h

```diff
@@ -76,7 +79,12 @@ typedef struct rti_common_t {
   scheduling_node_t** scheduling_nodes;
 
   // Number of scheduling nodes
-  int32_t number_of_scheduling_nodes;
+  uint16_t number_of_scheduling_nodes;
+
+  // Matrix of minimum delays between each nodes
+  // Rows represent upstreams and Columns represent downstreams.
+  // FOREVER_TAG means there is no path, ZERO_TAG means there is no delay.
```

Comment:
```suggestion
  // Matrix of minimum delays between pairs of nodes.
  // Rows represent upstream nodes and Columns represent downstream nodes.
  // FOREVER_TAG means there is no path, and ZERO_TAG means there is no delay.
  // This could be NULL if the matrix is not being used, so accesses should test for NULL first.
```